app/models/effective/datatables/broker_agency_datatable.rb

```diff
@@ -59,6 +59,10 @@ def nested_filter_definition
         }
 
       end
+
+      def authorized?(current_user, _controller, _action, _resource)
+        current_user.has_hbx_staff_role?
```

Comment:
```suggestion
        current_user&.person&.hbx_staff_role.present?
```

@TreyE Can you check if the instance of hbx_staff_role exists? The current method first checks if there is a string in an array which is the deprecated way of checking if a user has a role.